templates/api/knowledge-panels/health/ingredients/nova.tt.json

```diff
@@ -35,6 +35,8 @@
         [% END %]
     [% END %]
 [% END %]
+            <br>
```

Comment:
Adding <br> at the end of panels is probably not the right way, clients should decide how much spaces to leave.